secboot/secboot_sb.go

```diff
@@ -291,3 +296,32 @@ func DeleteKeys(node string, matches map[string]bool) error {
 
 	return nil
 }
+
+func GetPrimaryKeyHash(devicePath string, alg crypto.Hash) (salt []byte, digest []byte, err error) {
```

Comment:
This is returning HMAC rather than a hash, so the naming is a bit misleading. The same comment I made to `getPrimaryKeyDigest` also applies here.